assets/_sass/_keynotelist.scss

```diff
@@ -52,9 +65,12 @@ div.keynote-full {
 
 @media (max-width: 500px) {
 	section.keynotes-front {
-		a {
-			width: 32%;
-		height: 150px
+		div {
+			width:150px;
+		}
+		div.keynote-img {
```

Comment:
should probably be li.keynote-img with your changes.  The modifications here are because the original didn't look correct when I rescaled the browser.  Same or the change from 500px to 750px but I defer to your methods of testing it.